components/ai_chat/core/browser/utils.cc

```diff
@@ -178,6 +182,56 @@ const std::string& GetActionTypeQuestion(mojom::ActionType action_type) {
   return iter->second;
 }
 
+uint32_t GetMaxPageContentLengthFromContextSize(
+    std::optional<uint32_t> context_size) {
+  return static_cast<uint32_t>(
+      context_size.value_or(kCustomModelDefaultContextSize) *
+      kDefaultCharsPerToken);
```

Comment:
It is not very clear why this multiplication occurs based on the name of the function. I think this relationship should be documented with comments in the header declaration for this function.